### ISSUE TITLE
Date Picker doesn't work with JAWS

### ISSUE THREAD

**Issue by jkyoutsey** [Promact/md2#153]
Windows 10, JAWS, Chrome latest.
With JAWS enabled, the keyboard navigation is broken in the datepicker.
The focus will not move with the arrow keys.
What is read to the user is confusing.  It SOUNDS like it is reading all of the possible years from 1900 on for the year selector.
Just doesn't work for 508 at all.

ChromeVox seems to allow keyboard usage.  But I can't seem to get it to STOP reading the dropdown values (where space stops JAWS).  So, it's totally unusable in ChromeVox.

**Comment by dharmeshpipariya-zz** [comment]
@jkyoutsey we have updated datepicker component, update it in your project, check it and comment here is it working or not.
